aptos-move/block-executor/src/view.rs

```diff
@@ -1556,10 +1578,13 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> TModuleView
                     .borrow_mut()
                     .module_reads
                     .insert(state_key.clone());
-                state.unsync_map.fetch_module_data(state_key).map_or_else(
-                    || self.get_raw_base_value(state_key),
-                    |v| Ok(v.as_state_value()),
-                )
+                state
+                    .unsync_map
+                    .fetch_module_for_loader_v2(state_key)
```

Comment:
hm, above errors if we are using loader v2, but here the API is renamed to v2? in other words, is it just that the API was renamed and is used in the old flow, or do we actually go through this flow in sequential case even for v2? The latter seems bad, but I think it's the former, in which case maybe we should adjust the name to indicate it works in both cases?

Reply:
Good catch, I renamed incorrectly, should be `.._loader_v1`.  Fixed.